### ISSUE TITLE
Axis MinValue resets after changing series visibility or zoom 

### ISSUE THREAD

**Issue by LB767** [beto-rodriguez/Live-Charts#270]
I'm not exactly sure if some of this is actually intended behaviour but I was expecting that having a `MinValue` (or a `MaxValue`) set would force the axis to display between that range no matter what.

```xaml
<lvc:CartesianChart Grid.Row="2" Grid.Column="1" Grid.ColumnSpan="2" Margin="0,0,110,10" x:Name="Plot" Background="#FF161616" Visibility="Hidden" Zoom="X">
            <lvc:CartesianChart.AxisX>
                <lvc:Axis x:Name="PlotAxisX">
                    <lvc:Axis.Separator>
                        <lvc:Separator x:Name="PlotSeparator" StrokeThickness="1" >
                            <lvc:Separator.Stroke>
                                <SolidColorBrush Color="#404F56" />
                            </lvc:Separator.Stroke>
                        </lvc:Separator>
                    </lvc:Axis.Separator>
                </lvc:Axis>
            </lvc:CartesianChart.AxisX>
            <lvc:CartesianChart.AxisY>
                <lvc:Axis x:Name="PlotAxisY" MinValue="0">
                    <lvc:Axis.Separator>
                        <lvc:Separator StrokeThickness="1" StrokeDashArray="2">
                            <lvc:Separator.Stroke>
                                <SolidColorBrush Color="#404F56" />
                            </lvc:Separator.Stroke>
                        </lvc:Separator>
                    </lvc:Axis.Separator>
                </lvc:Axis>
            </lvc:CartesianChart.AxisY>
        </lvc:CartesianChart>
```

But apparently if only the `MinValue` property is set and the visibility of a series is changed, the axis range is modified. This however does not happen if both `MinValue` and `MaxValue` are set.

![visibility](https://cloud.githubusercontent.com/assets/20526358/16933896/eafa25fa-4d51-11e6-899f-5bd7057666f3.gif)

The axis also resets its bounds when zooming (here `Zoom="X"`) whether both Min and Max values are set or only one of the two.

![zooming](https://cloud.githubusercontent.com/assets/20526358/16933903/04244664-4d52-11e6-8a20-c6df730417bc.gif)

And a huge thanks for this library, I'm not an experienced programmer and the simplicity of these charts made my life incredibly easy!

**Comment by beto-rodriguez** [comment]
Hello, I am glad you find the library useful, that is the intention :smile: 

you were right about your observations, it was not working correctly, last commit should fix the observations you made.

I will release this fix in the next version (0.7.11) , thank you very much for this detailed report, it helps a lot.

I would appreciate if you could test it, and let me know if every thing makes sense now, if you could clone the library to test it would be awesome!, or wait for me to release next nuget version is ok too. it should not take too long, maye a couple of days.

Thanks again!

**Comment by LB767** [comment]
Tested it with the nuget version, works like a charm now. Thanks a lot!
